FabricExample/e2e/examplesTests/tests.e2e.ts

```diff
@@ -0,0 +1,111 @@
+import { device, expect, element, by } from 'detox';
+
+describe('Test432', () => {
+  beforeAll(async () => {
+    await device.reloadReactNative();
+  });
+
+  it('should Test432 exist', async () => {
+    await waitFor(element(by.id('root-screen-tests-Test432')))
+    .toBeVisible()
+    .whileElement(by.id('root-screen-examples-scrollview'))
+    .scroll(600, 'down', NaN, 0.85);
+
+    await expect(element(by.id('root-screen-tests-Test432'))).toBeVisible();
+    await element(by.id('root-screen-tests-Test432')).tap();
+  });
+
+  it('home-square should be fully visible', async () => {
+    await expect(element(by.id('home-square'))).toBeVisible(100);
+  });
+
+  it('squares from details screen should be fully visible', async () => {
```

Comment:
```suggestion
  it('headerRight elements should stay visible after toggling', async () => {
```

Same here & in many other places. I'm certain you see the pattern.

Reply:
2ed530f9171e919d9ad3722006170f54d0977fd7